types/git-init/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)

```

Comment:
I believe the repo content is hosted and released under same license already:https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE